lib/src/vector_math/vector4.dart

```diff
@@ -119,8 +119,8 @@ class Vector4 implements Vector {
 
   /// Returns a printable string
   @override
-  String toString() => '${_v4storage[0]},${_v4storage[1]},'
-      '${_v4storage[2]},${_v4storage[3]}';
+  String toString() => '[${_v4storage[0]},${_v4storage[1]},'
```

Comment:
The one in _64 also needs this fix

Reply:
it was also fixed in my original commit. Please double check it looks good.

Reply:
Ooh, completely missed it when reviewing on the phone. All good then! :)